go.mod

```diff
@@ -65,4 +65,5 @@ require (
 	k8s.io/api v0.18.6
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+	k8s.io/utils v0.0.0-20190221042446-c2654d5206da
```

Comment:
I wonder if this adds significantly to our binary size - we've had bad experiences with K8S dependencies ballooning the Podman executable by 5+ MB in size before...

Reply:
yeah, I know the pain, this is very small and without indirect dependencies. Is just 2 files but very useful helpers, at least we are using it a lot in k8s and ovn-kube

Reply:
forget to mention that we are going to need this for all the dual stack stuff
https://github.com/containers/podman/issues/7302

Reply:
I've removed dependency, if we need it later it can be revisited but the bits that we need does not justify vendoring